config.toml

```
@@ -58,6 +58,14 @@ description = "The CNCF Cloud Native Glossary Project is intended to be used as
 languageName ="English"
 # Weight used for sorting.
 weight = 1
+
+[languages.it]
+title = "Glossario Cloud Native"
+description = "Il progetto del Glossario CNCF Cloud Native è pensato per essere un riferimento per i termini comuni, usati nell'ambito delle applicazioni cloud native."
+languageName = "Italiano"
+# Weight used for sorting.
```

```suggestion
contentDir =  "content/it"
```
This is important parameter :) 

You can check the preview of the site (https://deploy-preview-404--cncfglossary.netlify.app/)
The preview does not show Italian contents.